boards/xtensa/intel_adsp_cavs25_tgph/intel_adsp_cavs25_tgph.yaml

```diff
@@ -0,0 +1,10 @@
+identifier: intel_adsp_cavs25_tgph
+name: CAVS 2.5 Audio DSP for Tiger Lake H PCH (Connected Audio Voice and Speech)
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+testing:
+  only_tags:
+     - kernel
+     - sof
```

Comment:
I did some cleanup here recently on the other cAVS boards, see #42748 and #42563. Please update to match

Reply:
Oops, yes, updated!